tests/test_generic_library.py

```diff
@@ -0,0 +1,137 @@
+import json
+from unittest import mock
+
+import boto3
+import pandas as pd
+from botocore.response import StreamingBody
+
+
+class MockContext:
+    aws_request_id = 999
+
+
+context_object = MockContext()
+
+
+def create_bucket(bucket_name):
```

Comment:
These could probably do with docstring and/or their own readme explaining what they are for and how to implement them.